source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -70,11 +77,39 @@ class NiRFmxWLANDriverApiTests : public Test {
     return TService::NewStub(device_server_->InProcessChannel());
   }
 
+  template <typename TAttr>
+  int32 get_attr_i32(const nidevice_grpc::Session& session, const std::string& selector_string, TAttr attribute_id)
+  {
+    auto response = client::get_attribute_i32(stub(), session, selector_string, attribute_id);
+    EXPECT_SUCCESS(session, response);
+    return response.attr_val();
+  }
+
  private:
   DeviceServerInterface* device_server_;
   std::unique_ptr<NiRFmxWLAN::Stub> stub_;
 };
 
+void GetCommaSeparatedStringFromArray(char* arrayOfNames, int numOfNames, int nameLength, char* commaSeparatedName)
+{
+   int i = 0;
+   int j=0;
```

Comment:
inconsistent spacing here. Also while we are at it I would recommend different variable names than ijk for readability

Reply:
Cleaned up the spacing. Realize I didn't have format on save turned on so now it should be in the right style throughout. Also, updated the names.